bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerChecker.java

```diff
@@ -195,10 +199,45 @@ private void verifyLedgerFragment(LedgerFragment fragment,
             bookieClient.readEntry(bookie, fragment
                     .getLedgerId(), firstStored, manycb, null);
         } else {
-            ReadManyEntriesCallback manycb = new ReadManyEntriesCallback(2,
+            assert lastStored > firstStored;
```

Comment:
let's not use `assert`. if `lastStored <= firstStored`, let's fail the callback `cb`.

Reply:
will change it.